src/utils/slackUtils.ts

```diff
@@ -0,0 +1,12 @@
+import { SlackWebhook } from '../types/slack';
+
+export const sendToSlack = ({ webhookEndpoint, data }: SlackWebhook) => {
+  const payload = {
+    method: 'POST',
+    body: JSON.stringify(data),
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  };
+  fetch(webhookEndpoint, payload);
```

Comment:
non-blocker) How about returning the value so we can handle it further?

Reply:
Thanks for the suggestion! Applied in [7ff0f58](https://github.com/exxocism/node-healthz-prober/pull/2/commits/7ff0f5827a4da169267226c56fa7e7c748e6a75b)